ivadomed/transforms.py

```diff
@@ -261,23 +264,66 @@ def __call__(self, sample, metadata=None):
 
         if len(zooms) == 2:
             zooms += [1.0]
-
-        hfactor = zooms[0] / self.hspace
-        wfactor = zooms[1] / self.wspace
-        dfactor = zooms[2] / self.dspace
+        if self.flag_pixel:
+            hfactor = self.hspace / sample.shape[0]
+            wfactor = self.wspace / sample.shape[1]
+            dfactor = self.dspace / sample.shape[2]
+        else:     
+            hfactor = zooms[0] / self.hspace
+            wfactor = zooms[1] / self.wspace
+            dfactor = zooms[2] / self.dspace
```

Comment:
Same comment here. I do not understand this different behavior you are implementing here.

Reply:
check above.

Reply:
See above.

Reply:
Check above.